test/network/BancorNetwork.ts

```diff
@@ -284,7 +326,7 @@ describe('BancorNetwork', () => {
         });
 
         it('should be properly initialized', async () => {
-            expect(await network.version()).to.equal(8);
+            expect(await network.version()).to.equal(9);
```

Comment:
can test the default rewards here as well

Reply:
added default rewards test